gcloud-java-dns/README.md

```diff
@@ -96,47 +93,53 @@ For other authentication options, see the [Authentication](https://github.com/Go
+exists within your project, you'll get a helpful error message telling you to choose another name. In the code below,
+replace "my_unique_zone" with a unique zone name. See more about naming rules [here](https://cloud.google.com/dns/api/v1/managedZones#name).*
+
+In this code snippet, we create a new zone in which we intend to
 manage DNS record for domain `someexampledomain.com.`
 
+*Important: The service may require that you verify ownership of the domain for which you are creating a zone.
+Hence, we recommend that you do so beforehand. You can verify ownership of
+a domain name [here](https://www.google.com/webmasters/verification/home). Note that Cloud DNS
+requires fully qualified domain names which must end with a period.*
+
 Add the following imports at the top of your file:
 
 ```java
-import com.google.gcloud.storage.Zone;
-import com.google.gcloud.storage.ZoneInfo;
+import com.google.gcloud.dns.Zone;
+import com.google.gcloud.dns.ZoneInfo;
 ```
 
 Then add the following code to create a zone.
 
-*Important: Zone names within each project have to be unique. If you choose a zone name that already
-exists, you'll get a helpful error message telling you to choose another name. In the code below,
-replace "my_unique_zone" with a unique zone name. See more about naming rules [here](https://cloud.google.com/dns/api/v1/managedZones).*
-
-*Important: The service may require that you verify ownership of the domain name that you are
-creating a zone for. Hence, we recommend that you do so beforehand. You can verify ownership of
-a domain name [here](https://www.google.com/webmasters/verification/home). Note that `gcloud-java`
-works with fully qualified domain names which must end with a period.*
-
 ```java
-// Create a zone metadata
+// Create a zone metadata object
 String zoneName = "my_unique_zone"; // Change this zone name which is unique within your project
 String domainName = "someexampledomain.com."; // Change this to a domain which you own
 String description = "This is a gcloud-java-dns sample zone.";
 ZoneInfo zoneInfo = ZoneInfo.of(zoneName, domainName, description);
 
 // Create zone in Google Cloud DNS
-Zone zone = dns.create(zoneInfo);
+Zone createdZone = dns.create(zoneInfo);
+System.out.printf("Zone was created and assigned ID %s.%n", createdZone.id());
 ```
 
 You now have an empty zone hosted in Google Cloud DNS which is ready to be populated with DNS
 records for domain name `someexampledomain.com.` Upon creating the zone, the cloud service
-automatically assigned a set of DNS servers to this zone which will be hosting the records, and
-created the default required SOA and NS records for the domain inside this zone. You can read the
-list of assigned DNS servers using accessor method `zone.nameServers()` and for instance print them
-on the screen as follows. You will need to add `import com.google.gcloud.dns.DnsRecord` to your
-source code.
+automatically assigned a set of DNS servers to host records for this zone, and
```

Comment:
Nit: I think "automatically" can be removed here, and you can also remove the "," before "and"